pkg/service/client.go

```diff
@@ -10,13 +10,13 @@ import (
 
 //DeleteAppFromServiceByRestAPI delete the app object in the service
 func DeleteAppFromServiceByRestAPI(serviceAPI string, id string, reqLogger logr.Logger ) error {
-	reqLogger.Info( "Calling REST API to DELETE app", "serviceAPI", serviceAPI, "App.id", id )
+	reqLogger.Info( "Calling REST Service to DELETE app", "serviceAPI", serviceAPI, "App.id", id )
```

Comment:
```suggestion
	reqLogger.Info( "Calling Service to DELETE app", "serviceAPI", serviceAPI, "App.id", id )
```

Is there a need to specify `REST`? This seems an implementation detail of the client, and if we swap out the current REST implementation in the client for something for something else, like if we changed to using GraphQL it seems we would need to change a lot of logging comments.

Reply:
The logs are important for we are able to troubleshoot it. 
As you can see the changes performed here is more to keep all texts equals and check the HTTP code status for we have sure if it worked or not as throw the errors.

Reply:
@camilamacedo86 I'm not sure you understood my point - can you re-read above comment

Reply:
Done as you suggested.